src/app/workerapp/cient-list/list-element/list-element.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngFor="let customer of custArr; let i = index" class="md:flex m-4 p-2 pl-4 pr-4 rounded-2xl bg-slate-200 text-slate-300 drop-shadow-md block items-center justify-between">
+    <div class="bg-slate-500 drop-shadow-md rounded-2xl p-2 pr-4 pl-4 mr-2">
+        {{custArr[i].customerName}}
+    </div>
+    <div class="bg-slate-500 drop-shadow-md rounded-2xl mt-3 mb-3 md:mt-0 md:mb-0 mr-2 p-2 pr-4 pl-4">
+        {{custArr[i].address}}
+    </div>
+    <div>
+        <div class="block w-fit md:flex md:w-80 md:justify-end">
+            <div class="bg-slate-500 drop-shadow-md rounded-2xl p-2 pr-4 pl-4 mr-2">
+                {{custArr[i].planningStatus}}
+            </div>
+            <button (click)="selectCustomer(i); this.parent?.parent?.goPage2();" class="bg-slate-500 drop-shadow-md rounded-2xl md:mt-0 mt-2 p-2 pr-4 pl-4">Go ></button>
```

Comment:
Instead of these functions, make separate components and routes like we saw in the angular course.